src/components/Calculator.js

```diff
@@ -0,0 +1,35 @@
+const Calculator = () => {
```

Comment:
- [ ] As it is a [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_components.md) for this project, " `Use class based components (All components should be class-based).` ", kindly ensure all you components are clased-based components not functional components.